src/frontend/components/actions/list.tsx

```diff
@@ -54,8 +54,10 @@ const List: React.FC<ActionProps> = ({ resource, setTag }) => {
     navigate({ search: search.toString() })
   }
 
+  const cssTag = `${resource.id}-list-action-content`
```

Comment:
Same here, you should also be able to use `action.name` to get the name of the action

Reply:
done